iceoryx_posh/test/moduletests/test_capro_message.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "iceoryx_posh/internal/capro/capro_message.hpp"
+#include "iceoryx_posh/internal/roudi/introspection/port_introspection.hpp"
+#include "test.hpp"
+
+using namespace ::testing;
+using namespace iox::capro;
+
+class CaproMessage_test : public Test
```

Comment:
As it is, we do not need a test fixture CaproMessage_test and could use TEST instead of TEST_F. (We generate all necessary data in the test itself and not in the test fixture). But we can keep it for future use (could be handy when the tests are be extended).

Reply:
okay